src/aks-preview/azext_aks_preview/tests/latest/recordings/test_aks_azure_service_mesh_with_egress_gateway.yaml

```diff
@@ -0,0 +1,3871 @@
+interactions:
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=118154459&view=results) to validate the change.
- test_aks_azure_service_mesh_with_egress_gateway

Please remove this recording file. As you've marked the case as live only, this file is useless.